docs/examples/PantheonAliases.php

```diff
@@ -111,6 +112,10 @@ private function getAliases() {
         if (isset($aliases[$key])) {
           break;
         }
+        if (!isset($environment->bindings)) {
```

Comment:
This is a good change to be made, but `$environment->bindings` will always exist since it's defined as an instance of the `Bindings` collection in the `Environment` constructor. Maybe this should be:
`if (empty($environment->bindings->all())) {`